### PR TITLE
tests: add storage nodes restart to payment tests

### PR DESCRIPTION
**Pull request by evgeniiz321**

### CODE REVIEW

File: pytest_tests/lib/helpers/node_management.py [hunk 1]
```diff
@@ -68,14 +68,19 @@ def stop_storage_nodes(nodes: list[StorageNode]) -> None:
...
+    The function restarts all storage nodes of the specified neofs env.
     Args:
-       nodes: the list of nodes to restart
+       neofs_env: NeoFSEnv instance
     """
-    stop_storage_nodes(nodes)
-    start_storage_nodes(nodes)
+    for node in neofs_env.storage_nodes:
+        node.stop()
+    time.sleep(5)
```
Comment (roman-khimov) [inline pytest_tests/lib/helpers/node_management.py:79]: Doesn't seem to be reliable.

File: pytest_tests/lib/helpers/node_management.py [hunk 2]
```diff
@@ -68,14 +68,19 @@ def stop_storage_nodes(nodes: list[StorageNode]) -> None:
...
     """
-    The function restarts specified storage nodes.
+    The function restarts all storage nodes of the specified neofs env.
     Args:
-       nodes: the list of nodes to restart
+       neofs_env: NeoFSEnv instance
     """
-    stop_storage_nodes(nodes)
-    start_storage_nodes(nodes)
+    for node in neofs_env.storage_nodes:
```
Comment (carpawell) [inline pytest_tests/lib/helpers/node_management.py:77]: can you please add some context to the commit? it is not clear to me why we need it and, moreover, why we cannot use `stop_storage_nodes(neofs_env.storage_nodes)`

File: pytest_tests/lib/helpers/node_management.py (hunk 1 above)
Comment (evgeniiz321) [reply pytest_tests/lib/helpers/node_management.py]: replaced with an explicit wait for the process termination

File: pytest_tests/lib/helpers/node_management.py (hunk 2 above)
Comment (evgeniiz321) [reply pytest_tests/lib/helpers/node_management.py]: Fixed. The only needed change here was to update `start_storage_nodes` function.

Comment (roman-khimov) [reply pytest_tests/lib/helpers/node_management.py]: It's not obvious from the commit message _why_ we're changing this. And it's just a workaround, not something critical to the test logic, so this better be explained for people reading the code in a year.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**